attributes/default.rb

```diff
@@ -57,6 +57,7 @@
 default['ntp']['keys'] = nil
 default['ntp']['trustedkey'] = nil
 default['ntp']['requestkey'] = nil
+default['ntp']['aci_support'] = nil
```

Comment:
Please rename this to `ntp['dscp']` as it is a more generic option than cisco aci only

Reply:
Thanks for pointing this out, I have done so.